pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1322,10 +1322,14 @@ func buildHTTPConnectionManager(listenerOpts buildListenerOpts, httpOpts *httpLi
 		filters = append(filters, xdsfilters.Alpn)
 	}
 
-	filters = append(filters, xdsfilters.Cors, xdsfilters.Fault)
+	filters = append(filters, xdsfilters.Cors)
 	filters = append(filters, listenerOpts.push.Telemetry.HTTPFilters(listenerOpts.proxy, listenerOpts.class)...)
 	filters = append(filters, xdsfilters.BuildRouterFilter(routerFilterCtx))
 
+	// Add fault filter at the top.
+	filters = append(filters[:1], filters[0:]...)
```

Comment:
Instead of this writing, i prefer 

```
filters := make([]*hcm.HttpFilter, len(httpFilters) + 1)
filters[0] = fault filter
copy(filters[1:], httpFilters)
```

Reply:
wouldn't this output duplicate filters？
append(filters[:1], filters[0:]...)

Reply:
Changed